### PR TITLE
Rename manifest.Create() to CreateApp

### LINKED ISSUE #207: Move application manifest to its own package
we need a package namespace for the pipeline manifest

### PR DESCRIPTION
**Pull request by hencrice**
This is necessary because we need a namespace for the pipeline manifest.

Fixes #207.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -87,9 +87,11 @@ github.com/konsorten/go-windows-terminal-sequences v1.0.1/go.mod h1:T0+1ngSBFLxv
 github.com/konsorten/go-windows-terminal-sequences v1.0.2 h1:DB17ag19krx9CFsz4o3enTrPXyIXCl+2iCXH/aMAp9s=
 github.com/konsorten/go-windows-terminal-sequences v1.0.2/go.mod h1:T0+1ngSBFLxvqU3pZ+m/2kptfBszLMUkC4ZK/EgS/cQ=
 github.com/kr/logfmt v0.0.0-20140226030751-b84e30acd515/go.mod h1:+0opPa2QZZtGFBFZlji/RkVcI2GknAs/DXo4wKdlNEc=
+github.com/kr/pretty v0.1.0 h1:L/CwN0zerZDmRFUapSPitk6f+Q3+0za1rQkzVuMiMFI=
```
Comment (efekarakus) [inline go.sum]: Why did we bring these additional dependencies?

File: internal/pkg/cli/init.go [hunk 2]
```diff
@@ -216,17 +216,17 @@ func (opts *InitAppOpts) createProject() error {
 }
 
 func (opts *InitAppOpts) createManifest() error {
-	manifest, err := manifest.Create(opts.AppName, opts.AppType)
+	manifest, err := app.Create(opts.AppName, opts.AppType)
```
Comment (efekarakus) [inline internal/pkg/cli/init.go]: Thinking out loud here 🤔 :

At first glance, `app.Create` is not as obvious as `manifest.Create` in terms of what it's creating.  
What do you think about keeping the `manifest` pkg as is, but changing the function `Create` to `CreateApp`. This way you can have `manifest.CreateApp` or `manifest.CreatePipeline`.

File: go.sum (hunk 1 above)
Comment (hencrice) [reply go.sum]: not sure why Go did it lol

File: internal/pkg/cli/init.go (hunk 2 above)
Comment (hencrice) [reply internal/pkg/cli/init.go]: yea kinda like the idea

**Pull request merged**